docs/source/awesome-trio-libraries.rst

```diff
@@ -0,0 +1,57 @@
+Awesome Trio Libraries
+======================
+
+.. List of Trio Libraries
+
+   A list of libraries that support Trio, similar to the awesome-python
+   list here: https://github.com/vinta/awesome-python/
+
+
+.. currentmodule:: trio
+
+You have completed the turorial, and are enthusiastic about building
+great new applications and libraries with async functionality.
+However, to get much useful work done you will want to use some of
+the great libraries that support trio-flavoured concurrency. This list
+is not complete, but gives a starting point. Another great way to find
+trio-compatible libraries is to search on PyPI for the `Framework :: trio`
+tag -> `PyPI Search <https://pypi.org/search/?q=Framework+%3A%3A+trio>`__
+
+
+Core Trio Libraries
+-------------------
+
+These libraries are part of the `python-trio Github team <https://github.com/python-trio>`__ and will be used in
+many trio projects:
+
+* `cookiecutter-trio <https://github.com/python-trio/cookiecutter-trio>`__ - Quickstart template for Trio projects
+* `pytest-trio <https://github.com/python-trio/pytest-trio>`__ - Pytest plugin for trio
+* `trio-typing <https://github.com/python-trio/trio-typing>`__ - Type hints for Trio and related projects
+* `sphinxcontrib-trio <https://github.com/python-trio/sphinxcontrib-trio>`__ - Make Sphinx better at documenting Python functions and methods
+* `trio-click <https://github.com/python-trio/trio-click>`__ - Python composable command line utility, trio-compatible version
+* `triopg <https://github.com/python-trio/triopg>`__ - PostgreSQL client for Trio based on asyncpg
+* `trio-mysql <https://github.com/python-trio/trio-mysql>`__ - Pure Python MySQL Client
+* `hypothesis-trio <https://github.com/python-trio/hypothesis-trio>`__ - Hypothesis plugin for trio
```

Comment:
I'm actually not sure what the status is of anything below `trio-click` here... maybe just file them under categories, instead of "core" libraries?

`trio-click` → "Building command line apps"
`triopg`, `trio-mysql` → already listed under databases
`hypothesis-trio` → "Testing"